common/kvstore/src/main/java/org/apache/spark/util/kvstore/KVTypeInfo.java

```diff
@@ -130,7 +130,7 @@ String getParentIndexName(String indexName) {
     Class<?> getType();
   }
 
-  private class FieldAccessor implements Accessor {
+  private static class FieldAccessor implements Accessor {
```

Comment:
static is better for inner classes that need no reference to the outer class. While I don't think it comes up -- those hidden references can cause GC or serialization issues in theory. Just cleaner.